extensions/void/package.json

```
@@ -272,7 +272,6 @@
     "@types/node": "^22.5.1",
     "@types/react": "^18.3.4",
     "@types/react-dom": "^18.3.0",
```

I wasn't able to build the extension without this. The fix should be to downgrade this version to `1.89.0` since that's the vscode engine version the project is using.

```suggestion
    "@types/react-dom": "^18.3.0",
    "@types/vscode": "^1.89.0",